conda_build/os_utils/macho.py

```diff
@@ -218,10 +218,10 @@ def get_id(path, build_prefix=None):
         return ''
 
 
-def get_rpaths(path):
+def get_rpaths(path, build_prefix=None):
     """Return a list of the dylib rpaths"""
-    res_pyldd = inspect_rpaths(path, resolve_dirnames=False, use_os_varnames=True)
-    return res_pyldd
+    dylib_loads = otool(path, build_prefix, is_rpath)
+    return [dylib_load['path'] for dylib_load in dylib_loads]
 
```

Comment:
why this change?

Reply:
`delete_rpath` and `insert_rpath` were both using `otool` instead of `pyldd`. Should this be reverted?

Reply:
Not sure, I'll check a bit later. In general I'm trying to move things towards our own implementations but let me get back to you on this. Thanks for all the effort you are all putting in to conda-build by the way. We at Anaconda greatly appreciate it.

Reply:
@mingwandroid any news on this ?

Reply:
Sorry not yet. I'll make sure to review it before the next release though. Thanks

Reply:
OK, I'm looking into the release now. It's grown in scope a lot, sorry about the delay @jschueller and @isuruf